apps/wellknown/views/application.py

```diff
@@ -13,6 +15,30 @@ class ApplicationListPagination(PageNumberPagination):
     max_page_size = 10000
 
 
+class LabelSlugFilter(CharFilter):
+    def filter(self, qs, value):
```

Comment:
After looking at the library code I believe it is treating these `Filter`s as presenting the same behaviors as a django Form Field. 

Try writing a `def validate(self, value)` to contain the slug validation logic.